### PR TITLE
Translate JN

### PR DESCRIPTION
**Pull request by laurani02**
Pontus, du var för snabb

## Describe your changes
Översatts JN:s text
## Checklist before requesting a review

- [x] I have performed a self-review of my changes
- [x] I have updated both Swedish and English pages (if not motivate why)

### CODE REVIEW

**Comment by psoder** [comment]
:)

**Review by Herkarl — APPROVED** [review]
Could perhaps have a link to [jml.datasektionen.se](https://jml.datasektionen.se). Although that page is only in Swedish, so it's fair not to.

Otherwise lgtm

File: en/clubs/jamlikhetsnamnden/body.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+# Equality Committee
 
-Nämnden som gör allt för att alla ska känna sig trygga och välkomna på Konglig Datasektionen!
+The club that does everything to make students feel safe and welcome at the CS chapter!
 
-Om du upptäcker något som du tycker är fel eller särbehandlande är det med oss du ska prata. Det kan vara allt från att en lärare säger opassande saker på en föreläsning, till att en student är taskig mot en sektionskamrat.
+If you discover something that you think is wrong, offensive or discriminates against any person or group, you should talk to us. It can be anything from a teacher saying inappropriate things in a lecture, to a student being mean to another.
 
-Så om du upptäcker något som är kränkande för någon person eller grupp, [maila Jämlikhetsnämndens ordförande](mailto:jno@datasektionen.se) – eller om det är en allmän fråga som du vill lyfta till diskussion i nämnden kan du [maila Jämlikhetsnämndens maillista](mailto:jamlikhet@d.kth.se).
+In the first place, you can contact the committee by emailing [the head of the Equality Board](mailto:jno@datasektionen.se).
```
Comment (psoder) [inline en/clubs/jamlikhetsnamnden/body.md:7]: "The easiest way to get in touch" är känns som en bättre formulering?

File: en/clubs/jamlikhetsnamnden/body.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
-# Jämlikhetsnämnden
+# Equality Committee
 
-Nämnden som gör allt för att alla ska känna sig trygga och välkomna på Konglig Datasektionen!
+The club that does everything to make students feel safe and welcome at the CS chapter!
 
-Om du upptäcker något som du tycker är fel eller särbehandlande är det med oss du ska prata. Det kan vara allt från att en lärare säger opassande saker på en föreläsning, till att en student är taskig mot en sektionskamrat.
+If you discover something that you think is wrong, offensive or discriminates against any person or group, you should talk to us. It can be anything from a teacher saying inappropriate things in a lecture, to a student being mean to another.
```
Comment (psoder) [inline en/clubs/jamlikhetsnamnden/body.md:5]: I
> in a lecture, to a student

tror jag inte att det ska vara något ",".

File: en/clubs/jamlikhetsnamnden/body.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
-Så om du upptäcker något som är kränkande för någon person eller grupp, [maila Jämlikhetsnämndens ordförande](mailto:jno@datasektionen.se) – eller om det är en allmän fråga som du vill lyfta till diskussion i nämnden kan du [maila Jämlikhetsnämndens maillista](mailto:jamlikhet@d.kth.se).
+In the first place, you can contact the committee by emailing [the head of the Equality Board](mailto:jno@datasektionen.se).
 
-Anmäl dig till Jämlikhetsnämndens maillista för att ta emot inbjudningar till nämndens möten etc, genom att fylla i detta formulär: [Anmälan](https://docs.google.com/forms/d/1_jXUk1pF4dCS-kWsv2FDjZYXZzuriaf4Qp_iGNzGIKE/viewform?usp=send_form)
+If you want to work on equality issues yourself, you are welcome to come to our meetings and to join our [discussion forum](https://www.facebook.com/groups/jmldiskussion/) We always post on our [Facebook page](https://www.facebook.com/jamlikhetsnamnden/) when we have meetings.
 
-Du kan även maila ordföranden på adressen ovan om du vill gå med i Jämlikhetsnämnden.
+You can also email the head of the committee if you want to join.
```
Comment (psoder) [inline en/clubs/jamlikhetsnamnden/body.md:11]: Det här stycket säger väl i princip samma sak som,

> In the first place, you can contact the committee by emailing [the head of the Equality Board](mailto:jno@datasektionen.se).

Comment (psoder) [inline en/clubs/jamlikhetsnamnden/body.md:11]: Det här stycket säger väl i princip samma sak som,

> In the first place, you can contact the committee by emailing [the head of the Equality Board](mailto:jno@datasektionen.se).

File: en/clubs/jamlikhetsnamnden/body.md (hunk 2 above)
Comment (psoder) [inline en/clubs/jamlikhetsnamnden/body.md:5]: I
> in a lecture, to a student

tror jag inte att det ska vara något ",".

File: en/clubs/jamlikhetsnamnden/body.md (hunk 1 above)
Comment (psoder) [inline en/clubs/jamlikhetsnamnden/body.md:7]: "The easiest way to get in touch" är känns som en bättre formulering?

**Review by psoder — CHANGES_REQUESTED** [review]

**Comment by laurani02** [comment]
Har uppdaterat nu :)

**Review by psoder — APPROVED** [review]
lgtm

**Pull request merged**